.yarnrc.yml

```diff
@@ -0,0 +1,7 @@
+nodeLinker: node-modules
```

Comment:
This keeps Yarn's behavior compatible with the `node_modules` layout used by yarn v1 and all versions of npm.